index.html

```diff
@@ -977,7 +969,8 @@ <h4>Samuel Dobson</h4>
       <div class="container">
         <div class="row justify-content-center">
           <div class="col-md-4">
-		  <span class="copyright">Copyright &copy; <a href="https://chaincode.com">Chaincode Labs</a> 2020</span>
+            <span class="copyright">Copyright &copy; <a href="https://chaincode.com">Chaincode Labs</a> 2020</span>
+            <a href="residency@chaincode.com">residency@chaincode.com</a>
```

Comment:
This shouldn't be a link. It's an email. Right now that would click through to residency.chaincode.com/residency@chaincode.com.

I'd prefer a span as found on the resources page.

Reply:
done